index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
```

Comment:
We already include the FA CSS in the page. Do we actually need the JS?

Reply:
Hi, I tried it without the JS, but it displays empty boxes when you open it on the browser. I'll attach a screenshot..

Reply:
![without_js](https://user-images.githubusercontent.com/32118977/36983708-1673b16a-20b9-11e8-9724-acb8dac02af3.png)

Reply:
Going from [the documentation here](https://fontawesome.com/how-to-use/web-fonts-with-css), it should be possible to use font awesome with CSS only.
I might have given the wrong names in #426. We currently use font-awesome 4 on the landing page, but the links are for the font-awesome 5 names. You can try out [the names for fa 5](https://fontawesome.com/v4.7.0/icon/bar-chart/) instead.

Reply:
Oh, okay